csharp/src/Google.Protobuf/JsonFormatter.cs

```diff
@@ -879,6 +880,10 @@ private static class OriginalEnumValueHelper
             private static readonly Dictionary<System.Type, Dictionary<object, string>> dictionaries
                 = new Dictionary<System.Type, Dictionary<object, string>>();
 
+#if NET5_0_OR_GREATER
```

Comment:
Would it be better to add internal definition of the trimming attributes and the enum to the project so that these ifdefs are not needed? (It is what we do in dotnet/runtime.)